pkgs/by-name/ch/chart-releaser/package.nix

```diff
@@ -0,0 +1,67 @@
+{ buildGoModule
+, coreutils
+, fetchFromGitHub
+, git
+, installShellFiles
+, kubernetes-helm
+, lib
+, makeWrapper
+}:
+
+buildGoModule rec {
+  pname = "chart-releaser";
+  version = "1.6.1";
+
+  # 0102fa30dd78df0d0f8093c4607e6080972ff82d
+
+  src = fetchFromGitHub {
+    owner = "helm";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-8+O9JErEB1Z/zlrWm975v5Qf0YG0lbPcjY5LlDKw8U4=";
+
+  };
+
+  # Don't run tests.
+  doCheck = false;
+  doInstallCheck = false;
+
+  vendorHash = "sha256-S/V1kTgD/cXaJNYpPPNjM9ya2zv6Bsy/YBn7I/1EjwI=";
+
+  postPatch = ''
+    substituteInPlace pkg/config/config.go \
+      --replace "\"/etc/ct\"," "\"$out/etc/ct\","
+  '';
+
+  # https://github.com/helm/chart-releaser/blob/fa01315c4668d4fca627a5afc67409e31b27305c/.goreleaser.yml#L37
+  ldflags = [
+    "-w"
+    "-s"
+    "-X github.com/helm/chart-releaser/v3/ct/cmd.Version=${version}"
+    "-X github.com/helm/chart-releaser/v3/ct/cmd.GitCommit=${src.rev}"
```

Comment:
> nixpkgs on  feat/chart-releaser [!+] ✦ ➜  result/bin/cr version
GitVersion:  unreleased
GitCommit:   unknown
BuildDate:   unknown
GoVersion:   go1.22.2
Compiler:    gc
Platform:    linux/amd64
License:     Apache 2.0

Looks like it does not embed those string properly?